app/controllers/admin/invoices_controller.rb

```diff
@@ -1,7 +1,20 @@
 class Admin::InvoicesController < ApplicationController
   def index
+    @invoices = Invoice.all.order(:id)
   end
 
-  def show 
+  def show
+    @invoice = Invoice.find(params[:id])
+  end
+
+  def update
+    invoice = Invoice.find(params[:id])
+    invoice.update(invoice_items_params)
+    redirect_to admin_invoice_path(invoice)
+  end
+
+  private
+  def invoice_items_params
```

Comment:
this is minor but I think we're working with invoices rather than invoice_items here